pkg/northbound/diags/diags_new_test.go

```diff
@@ -100,17 +101,17 @@ func Test_ListNetworkChanges(t *testing.T) {
 
 	mockNwChStore, ok := mgrTest.NetworkChangesStore.(*mockstore.MockNetworkChangesStore)
 	assert.Assert(t, ok, "casting mock store")
-	mockNwChStore.EXPECT().List(gomock.Any()).DoAndReturn(func(ch chan<- *networkchangetypes.NetworkChange) error {
+	mockNwChStore.EXPECT().List(gomock.Any()).DoAndReturn(func(ch chan<- *networkchangetypes.NetworkChange) (stream.Context, error) {
 		// Send our network changes as a streamed response to store List()
 		go func() {
 			for _, nwch := range networkChanges {
 				ch <- nwch
 			}
-			// Finally send an empty one like the real List() does
-			ch <- nil
```

Comment:
@SeanCondon FYI the store does not send a `nil` value on the channel when it's finished. It just closes the channel, and a channel read returns `nil` when the channel is closed.

Reply:
ok